toolchain/check/convert.cpp

```diff
@@ -991,7 +991,24 @@ static auto PerformBuiltinConversion(Context& context, SemIR::LocId loc_id,
   }
 
   if (sem_ir.types().Is<SemIR::FacetType>(target.type_id)) {
-    if (sem_ir.types().Is<SemIR::FacetType>(value_type_id)) {
+    auto lookup_inst = value_id;
```

Comment:
We'd usually call this `lookup_inst_id` instead to indicate that it's an `InstId` not an `Inst`.

Reply:
Right, done. Thanks for the review!